### PR TITLE
Add sage images for AD

### PR DESCRIPTION
**Pull request by cg2wilson**
Sage images for AD added!

### CODE REVIEW

File: source/calculus/source/03-AD/08.ptx [hunk 1]
```diff
@@ -29,18 +29,252 @@
...
+                    geometry += sll2
+                    geometry += label_side
+
+                    # Set axes and title
+                    geometry.set_axes_range(-1, side_large_square + 1, -1, side_large_square + 1)
+                    geometry.axes(False)
+
+                    # Show the plot
+                    geometry
+                  </sageplot>
```
Comment (StevenClontz) [inline source/calculus/source/03-AD/08.ptx:90]: Old:

image

New:

image

I think we should increase the font size in the new one a bit

File: source/calculus/source/03-AD/08.ptx [hunk 2]
```diff
@@ -199,7 +433,30 @@
...
+              geometry += front
+              geometry += front_border
+              geometry += back
+              geometry += back_border
+              geometry += right
+              geometry += right_border
+              geometry.axes(False)
+              geometry
+            </sageplot>
+          </image>
```
Comment (StevenClontz) [inline source/calculus/source/03-AD/08.ptx:459]: Old:

image

New:

image

Maybe it's just me, but I think its easier to understand the image when the darker side is on bottom.

Comment (StevenClontz) [reply source/calculus/source/03-AD/08.ptx:459]: If possible, using dashes for sides obscured to the viewer might help as well.

File: source/calculus/source/03-AD/08.ptx (hunk 1 above)
Comment (cg2wilson) [reply source/calculus/source/03-AD/08.ptx]: i increased the font size to 15pt- i can go larger if people think it's necessary.

File: source/calculus/source/03-AD/08.ptx (hunk 2 above)
Comment (cg2wilson) [reply source/calculus/source/03-AD/08.ptx]: the most recent commit changes the shading (and doesn't remove the helper axes, oops), and i'm working on adding the dashed lines

**Review by StevenClontz — APPROVED** [review]

**Pull request merged**